src/main/java/io/cdap/e2e/utils/ConstantsUtil.java

```diff
@@ -93,4 +96,9 @@ public class ConstantsUtil {
    * PIPELINE_RUN_TIMEOUT_SECONDS: To be used as a timeout for Pipeline Runs
    */
   public static final int PIPELINE_RUN_TIMEOUT_SECONDS = 900;
+
+  /**
+   * MEDIUM_TIMEOUT_SECONDS: To be used as a wait for retry
+   */
+  public static final int MEDIUM_TIMEOUT_SECONDS = 10;
```

Comment:
Looking at the usage would not SMALL_TIMEOUT_SECONDS work ?

Reply:
It works on local execution, used  ``SMALL_TIMEOUT_SECONDS``` instead of medium everywhere for now.  ()

Note : There is a possibility that due to some external delay in the Login Flow a small timeout may cause some flakiness.

(https://github.com/cdapio/cdap-e2e-tests/pull/197/commits/44be5d5c5c9bf2c3ee877d3eb7dd3aa43c414929#diff-653a26322c5148b982b3df96f0640a19488ea600b8501ce2695a5218dda8be2bR45)